vm/array.go

```diff
@@ -77,7 +77,7 @@ func initializeArray(elements []Object) *ArrayObject {
 	return &ArrayObject{Elements: elements, Class: arrayClass}
 }
 
-func init() {
+func initArray() {
```

Comment:
OK, I understood that using `init()` is intentional 😺 
I just thought it looked inconsistent with other initialization functions.